manifests/k/Keboola/KeboolaCLI/2.1.1/Keboola.KeboolaCLI.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Keboola.KeboolaCLI
+PackageVersion: 2.1.1
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64

```

Comment:
```suggestion
  Architecture: x64
  Scope: machine
```
```
C:\Program Files\Keboola CLI
```

Reply:
Oh, I see. I missed that option totally. Thank you.